analysis_driver/dataset.py

```diff
@@ -31,6 +29,7 @@ def __init__(self, name, most_recent_proc=None):
         self.name = name
         self.most_recent_proc = MostRecentProc(self.type, self.name, most_recent_proc)
         self._ntf = None
+        self._lims_ntf = None
```

Comment:
Does this need to be in `Dataset` if it is only going to be used in the `SampleDataset`